pages/common/docker-rm.md

```diff
@@ -0,0 +1,20 @@
+# docker rm
+
+> Remove one or more containers.
+> More information: <https://docs.docker.com/engine/reference/commandline/rm>.
+
+- Show help:
+
+`docker rm`
+
+- Remove containers:
+
+`docker rm {{container1 container2 ...}}`
+
+- Force remove a container:
+
+`docker rm --force {{container1 container2 ...}}`
+
+- Remove container and its volumes:
+
+`docker rm --volumes {{container}}`
```

Comment:
```suggestion
- Remove containers:

`docker rm {{container1 container2 ...}}`

- Force remove a container:

`docker rm --force {{container1 container2 ...}}`

- Remove a container and its volumes:

`docker rm --volumes {{container}}`

- Display help:

`docker rm`
```

Similar to https://github.com/tldr-pages/tldr/pull/11127/files#r1367765109

Ref.

```zsh
➜  ~ sudo docker rm
Place your right index finger on the fingerprint reader
"docker rm" requires at least 1 argument.
See 'docker rm --help'.

Usage:  docker rm [OPTIONS] CONTAINER [CONTAINER...]

Remove one or more containers
➜  ~ sudo docker rm --help

Usage:  docker rm [OPTIONS] CONTAINER [CONTAINER...]

Remove one or more containers

Aliases:
  docker container rm, docker container remove, docker rm

Options:
  -f, --force     Force the removal of a running
                  container (uses SIGKILL)
  -l, --link      Remove the specified link
  -v, --volumes   Remove anonymous volumes associated
                  with the container
```